addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -8008,6 +8015,75 @@ registry.VersionControl = SnippetOptionWidget.extend({
         });
         return this._super(...arguments);
     },
+
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+
+    /**
+     * @override
+     */
+    async updateUI() {
+        // Waiting for the options to be visible so the alert can be correctly
+        // displayed.
+        this.options.wysiwyg.waitForEmptyMutexAction().then(() => {
+            const alertEl = this.$el[0].querySelector("we-alert");
+            const optionSectionEl = this.$el[0].parentElement;
+            if (!alertEl || optionSectionEl.classList.contains("o_we_outdated_block_options")) {
+                return;
+            }
+            // Adding a class to hide the other options, to only have the alert
+            // displayed (so it is placed correctly).
+            optionSectionEl.classList.add("o_we_outdated_block_options");
+        });
+        return this._super(...arguments);
+    },
```

Comment:
@sobo-odoo This diff isn't equivalent to simply have this in CSS:

```diff
- &.o_we_outdated_block_options {
+ &:has(.o_outdated_block) {
    // Outdated block options
    padding: 0 !important;

    > we-customizeblock-option {
        &:not(.snippet-option-VersionControl) {
            display: none !important;
        }

        &.snippet-option-VersionControl {
            // Outdated snippet alert
            padding: 0 !important;
        }
    }
}
```

Reply:
`:has` is not yet supported enough to use it: https://caniuse.com/?search=%3Ahas%20css
Also, even when it will, I don't think it will be a nice practice to use it (but we'll see).